__tests__/groups/group.test.js

```diff
@@ -243,4 +243,18 @@ describe('Discord Groups Page', () => {
 
     expect(groupCreationModalClosed).toBeFalsy();
   });
+
+  test('Should display only specified groups when dev=true and name=<group-name> with different case', async () => {
```

Comment:
Add negative test cases that confirm groups not matching the search query are not displayed.